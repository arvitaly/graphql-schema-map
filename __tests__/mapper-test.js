"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const __1 = require("./..");
const schema_1 = require("./../__fixtures__/schema");
class Mapper {
    mapGraphQLSchema(config) {
        return {
            query: config.query,
            mutation: config.mutation,
        };
    }
    mapGraphQLObjectType(config) {
        return {
            fields: config.fields.map((f) => {
                return f;
            }),
        };
    }
    mapGraphQLObjectTypeField(config) {
        return config.name;
    }
}
describe("Mapper", () => {
    it("simple", () => {
        const result = __1.default(schema_1.default, new Mapper());
        expect(result).toMatchSnapshot();
    });
});
