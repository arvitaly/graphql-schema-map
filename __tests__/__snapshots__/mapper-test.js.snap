// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mapper simple 1`] = `
Object {
  "mutation": undefined,
  "query": Object {
    "fields": Array [
      "node",
      "viewer",
    ],
  },
}
`;
